packages/experience-builder-sdk/src/blocks/VisualEditor.tsx

```diff
@@ -1,9 +1,27 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { InternalSDKMode } from '../types';
-import { componentRegistry } from '../core/componentRegistry';
-import VisualEditor from '@contentful/experience-builder-visual-editor';
 import { VisualEditorContextProvider } from './editor/VisualEditorContext';
 
+const version = '0.0.1-pre-20231212T210325.0';
```

Comment:
This variable can be any semver tag or version, for example:
```ts
const version = '^0.0.1-pre';
```

Reply:
Yeah this is a great start. In the future I think we can potentially look at having the user_interface communicate the build version via post message so there becomes no reliance on users updating their SDK to get the most recent version of the visual editor.

Definitely not a requirement right now